packages/react-core/src/components/ToggleGroup/ToggleGroup.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/ToggleGroup/toggle-group';
+import ToggleGroupContext from './ToggleGroupContext';
+import { Divider } from '../Divider';
+
+export enum ToggleGroupVariant {
+  default = 'default',
+  light = 'light'
+}
+
+export interface ToggleGroupProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the toggle group */
+  children?: React.ReactNode;
+  /** Additional classes added to the toggle group */
+  className?: string;
+  /** Adds toggle group variant styles */
+  variant?: ToggleGroupVariant | 'light' | 'default';
+  /** Accessible label for the toggle group */
+  'aria-label'?: string;
+}
+
+export class ToggleGroup extends React.Component<ToggleGroupProps> {
```

Comment:
This could also be a functional component

Reply:
even as a context provider?

Reply:
Yes, it's fine if you want to leave it as a class component tho

Reply:
I made that change with the others. 
Love learning new things - I gotta stop assuming i guess :)